clowdapp.yaml

```diff
@@ -46,10 +46,11 @@ objects:
         - command:
             - bash
             - -c
-            - |
-              for i in {1..120}; do sleep 1;
```

Comment:
This 120 cycle limit was put in to leverage backoff and aid for failed job runs. Can we keep some kind of limit other than infinite loop?
Have a look at convo at https://github.com/RedHatInsights/compliance-backend/pull/1012#discussion_r712831489

Reply:
@vkrizan  check: https://github.com/RedHatInsights/compliance-backend/pull/1220/commits/efad82845a795c998a99dc8ac0fe78c817980446

Reply:
Thanks Victor. 
I don't find the option to be essential.

Reply:
I find it a nice to have, and quite honestly, being able to fine tune it gives us the advantage of adjust it based on the environment we're deploying to - i.e. -> on a fresh deployment, it is reasonable to extend the timeout, as it is expected most dependencies won't be ready right away. 

On the contrary, on a long living environment, a deployment should take less time and dependent services are likely to be available so we can cut that timeout down and crash earlier if there's a real ongoing problem.